### ISSUE TITLE
Sony Playstation Cheats in Wrong Format

### ISSUE THREAD

**Issue by christopher-s** [libretro/libretro-database#414]
A recent commit has re-enabled cheat support in beetle-psx, however all of the PSX cheats in this database are of an incorrect format.

This is the commit in question:
https://github.com/libretro/beetle-psx-libretro/commit/272f5c2e0cf4a90ea620b7584e50d0e38683c736

It just uses the Mednafen code for cheat support and this doesn't allow for "+" (plus) symbols for the cheat codes.

For example, a cheat code in this database would look like:
cheat0_code = "8009D970+03E7"

But, the beetle-psx code will throw an error due to the plus symbol.

Instead it should read:
cheat0_code = "8009D97003E7"

I've tested this myself and it appears to work with removing the plus symbol.

The solution might be to adjust all the cheat files (likely doing a regex search/replace), or adjust the beetle-psx cheat decoding so it ignores the plus.

**Comment by RobLoach** [comment]
```
cheat5_desc = "Infinite vitality gauge P1 & P2"
cheat5_code = "D011144C+00D6+3011144C+0000"
cheat5_enable = false 
```

**Comment by christopher-s** [comment]
Yeah - right now, I have no idea how those additional use cases would be handled with the Mednafen cheat format.

There's very little information on it.

**Comment by RobLoach** [comment]
Is this still an issue? We could likely parse the string on the libretro core side and strip the + signs out.

**Comment by christopher-s** [comment]
I haven't looked at this in ages. When I have a moment over the next week I'll check.

**Comment by Starlightbotanist** [comment]
Looks like this is good to close. Was fixed in [2613b5f
](https://github.com/libretro/beetle-psx-libretro/commit/2613b5fefe156360252d3e11dec0c9c884d6a6d7).

I tested it to be sure, and cheats do work fine that contain "+"s.
